appcli/orchestrators.py

```diff
@@ -297,15 +423,65 @@ def decrypt_file(encrypted_file: Path, key_file: Path):
         key_file (Path): Key to use for decryption.
 
     Returns:
-        [type]: Path to the decrypted file.
+        Path: Path to the decrypted file.
     """
     if not key_file.is_file():
         logger.info(
             "No decryption key found. [%s] will not be decrypted.", encrypted_file
         )
         return encrypted_file
 
-    logger.info("Decrypting file [%s] using [%s].", str(encrypted_file), key_file)
+    logger.debug("Decrypting file [%s] using [%s].", str(encrypted_file), key_file)
     decrypted_file: Path = Path(NamedTemporaryFile(delete=False).name)
     crypto.decrypt_values_in_file(encrypted_file, decrypted_file, key_file)
     return decrypted_file
+
+
+def execute_compose(
```

Comment:
Looks like this is only applicable to the docker compose orchestrator. I think it belongs in there as a private method (`__execute_compose`) rather than out here

Reply:
No it's also applicable to the docker swarm orchestrator for running tasks, hence why I've pulled it out deliberately.